src/detect.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+
+
+def _get_card_definitions(dir_):
+    return {
+        i.replace(".json", ""): json.load(open(os.path.join(dir_, i)))
+        for i in next(os.walk(dir_))[2]
+        if i.endswith(".json")
+    }
+
+
+script_dir = os.path.dirname(os.path.realpath(__file__))
+
```

Comment:
this code wont be needed if all the jsons are in one file

Reply:
correct, it's removed now.